### PR TITLE
Adding helm exit code output variable

### PR DESCRIPTION
**Pull request by shigupt202**
Fixes https://github.com/microsoft/azure-pipelines-tasks/issues/12532

### CODE REVIEW

File: Tasks/HelmDeployV0/src/kubernetescli.ts [hunk 1]
```diff
@@ -29,9 +29,11 @@ export default class kubernetescli extends basecommand {
     public setKubeConfigEnvVariable() {
         if (this.kubeconfigPath && fs.existsSync(this.kubeconfigPath)) {
             tl.setVariable("KUBECONFIG", this.kubeconfigPath);
+            tl.setVariable('helmExitCode', tl.TaskResult.Succeeded.toString());
```
Comment (ds-ms) [inline Tasks/HelmDeployV0/src/kubernetescli.ts:32]: Use a different enum. or just have global variables for `0` and `2` right now. 
This doesn't explicitly imply that it's going to result in 0 or 2

Comment (shigupt202) [reply Tasks/HelmDeployV0/src/kubernetescli.ts:32]: We are using tl.TaskResult.Succeeded in other parts of the task as well. I thought we should keep the codes used for Success and Failure consistent, that's why I'm using the same thing here as well.

**Comment by shigupt202** [comment]
@ds-ms There's no test setup for this task.

**Comment by shigupt202** [comment]
> Add test

There's no test setup for this task.

**Pull request merged**